srfi-119/wisp-guile.scm

```diff
@@ -1,46 +1,25 @@
-;; an SRFI. Only change it, if you agree to this possible relicensing
-;; of your contribution to this file. I will not accept changes here
-;; which do not allow that.
-
-; we need to be able to replace end-of-line characters in brackets and strings
-
-;; TODO: Check whether I can offload the string processing to the
-;; read-function. That’s a source of endless complications. Required:
-;; A kind of unrolling step which appends the string-representation of
-;; the read strings back into the code. I would have to process a list
-;; of strings instead of one big string. Or rather, each line would be
-;; a list of strings.
-
-;; bootstrap via python3 wisp.py wisp-guile.w > 1 && guile 1 wisp-guile.w > 2 && guile 2 wisp-guile.w > 3 && diff 2 3
-;; 
-;; -Author: Arne Babenhauserheide
-
-;; Copyright (C) Arne Babenhauserheide (2013--2015). All Rights Reserved.
-
-;; Permission is hereby granted, free of charge, to any person
-;; obtaining a copy of this software and associated documentation
-;; files (the "Software"), to deal in the Software without
-;; restriction, including without limitation the rights to use, copy,
-;; modify, merge, publish, distribute, sublicense, and/or sell copies
-;; of the Software, and to permit persons to whom the Software is
-;; furnished to do so, subject to the following conditions:
-;; 
-;; The above copyright notice and this permission notice shall be
-;; included in all copies or substantial portions of the Software.
-;; 
-;; THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
-;; EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
-;; MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
-;; NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
-;; BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
-;; ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
-;; CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-;; SOFTWARE.
-
+;;;; SPDX-FileCopyrightText: 2013 - 2015 Arne Babenhauserheide <arne_bab@web.de>
+;;;;
+;;;; SPDX-License-Identifier: MIT
+;;;;
+;;;; This file might need to be licensed permissively for inclusion in
+;;;; an SRFI. Only change it, if you agree to this possible relicensing
+;;;; of your contribution to this file. I will not accept changes here
+;;;; which do not allow that.
+;;;;
+;;;; we need to be able to replace end-of-line characters in brackets and strings
+;;;;
+;;;; TODO: Check whether I can offload the string processing to the
+;;;; read-function. That’s a source of endless complications. Required:
+;;;; A kind of unrolling step which appends the string-representation of
+;;;; the read strings back into the code. I would have to process a list
+;;;; of strings instead of one big string. Or rather, each line would be
+;;;; a list of strings.
+;;;;
+;;;; bootstrap via python3 wisp.py wisp-guile.w > 1 && guile 1 wisp-guile.w > 2 && guile 2 wisp-guile.w > 3 && diff 2 3
```

Comment:
why does this change to four semicolons for the TODO?

Reply:
I previously had the reuse tool use ';;;;' for its SINGLE_LINE comment; that's now been changed to ';;;', per discussions on the scheme-discuss list.